src/main/java/bio/terra/cli/command/config/set/Resources.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.cli.command.config.set;
+
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.context.GlobalContext;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the fourth-level "terra config set resources" command. */
+@Command(
+    name = "resources",
+    description = "Set the maximum number of resources allowed per workspace.")
+public class Resources extends BaseCommand {
+
+  @CommandLine.ArgGroup(exclusive = true, multiplicity = "1")
+  ImageIdArgGroup argGroup;
+
+  static class ImageIdArgGroup {
+    @CommandLine.Option(
```

Comment:
Can you restrict this to be greater than zero? Otherwise you need a good way to describe/display zero resources.

Reply:
Good catch. Added validation that `maxResources > 0`.